demos/eigenvalues_QG_basinmodes/qgbasinmodes.rst

```diff
@@ -0,0 +1,209 @@
+Oceanic Basin Modes: Quasi-Geostrophic approach
+===============================================
+
+Contributed by Christine Kaufhold and Francis Poulin.
+
+As a continuation of the Quasi-Geostrophic (QG) model described in the
+other tutorial, we will now see how we can use firedrake to compute
+the spatial structure and frequencies of the freely evolving modes in this system, what are referred at as basin modes.
+Oceanic basin modes are low frequency structures that propagate
+zonally in the oceans that alter the dynamics of Western Boundary Currents,
+such as the Gulf Stream. In this particular tutorial we will show how to
+solve the QG eigenvalue problem with no basic state and no dissipation.
+forces.
+Unlike the other demo that integrated the equations forward in time, in this
+probelm it is necessary to compute the eigenvalues and eigenfunctions
+for a particular differential operator. This requires using PETC matrices
+and eigenvalues solvers in SLEPc.
+
+
+Governing PDE
+=============
+
+We will first present the nonlinear, one-layer QG equation that was used
+in the other tutorial (link). The interested reader can find the
+derivations in Pedlosky:1992 and Vallis:2006.
```

Comment:
To make these into actual citations, we should just copy the bib file from the other demo directory (we don't have a better scheme right now), and add:

```
.. rubric:: References

.. bibliography:: qg_refs.bib
```

At the foot of this file.

Then you can produce citations by writing

```
derivations in :cite:`Pedlosky:1992` and :cite:`Vallis:2006`.
```